app/views/community/index.html.slim

```diff
@@ -37,8 +35,6 @@ table.users.table.table-striped.table-bordered.table-condensed.table-hover
         td = link_to_user_roles(user)
         td = user.country
         td = user.location
-        td = time_for_user(user)
```

Comment:
I think™ with this column no longer in place we may even be able to fully get rid of the `time_for_user` view helper. Can you please double check that?

Reply:
Indeed, it was only referenced in this table. I will remove the helper method and associated spec.